esmvaltool/recipes/model_evaluation/recipe_model_evaluation_portraits.yml

```diff
@@ -0,0 +1,535 @@
+             tier: 2, start_year: 2001, end_year: 2009, <<: *ref_tag}
+      pr: &var_rmse
+        variable: pr
+        <<: *var_settings
+        preprocessor: rmse
+        <<: *era5_ref
+      pr_2:
+        variable: pr
+        short_name: pr
+        <<: *var_rmse
+        split: Ref2
+        additional_datasets:
+          - {dataset: GPCP-SG, project: OBS, type: atmos, version: 2.3, tier: 2,
+             <<: *ref_tag}
+      psl:
+        variable: psl
+        <<: *var_rmse
+        <<: *era5_ref
+      rlut:
+        variable: rlut
+        <<: *var_rmse
+        <<: *era5_ref
+      rlut_2:
+        variable: rlut
+        short_name: rlut
+        <<: *var_rmse
+        split: Ref2
+        additional_datasets:
+          - {dataset: CERES-EBAF, project: OBS, type: sat, version: Ed4.2,
+             tier: 2, start_year: 2001, end_year: 2009, <<: *ref_tag}
+      rsut:
+        <<: *var_rmse
+        variable: rsut
+        additional_datasets:
+          - {dataset: ISCCP-FH, project: OBS, type: sat, version: v0,
+             tier: 2, start_year: 2000, end_year: 2009, <<: *ref_tag}
+      rsut_2:
+        variable: rsut
+        short_name: rsut
+        <<: *var_rmse
+        split: Ref2
+        additional_datasets:
+          - {dataset: CERES-EBAF, project: OBS, type: sat, version: Ed4.2,
+             tier: 2, start_year: 2001, end_year: 2009, <<: *ref_tag}
+      swcre:
+        variable: swcre
+        <<: *var_rmse
+        derive: true
+        force_derivation: true
+        channel: Amon
+        additional_datasets:
+          - {dataset: ISCCP-FH, project: OBS, type: sat, version: v0,
+             tier: 2, start_year: 2000, end_year: 2009, <<: *ref_tag}
+      swcre_2:
+        variable: swcre
+        short_name: swcre
+        <<: *var_rmse
+        channel: Amon
+        derive: true
+        force_derivation: true
+        split: Ref2
+        additional_datasets:
+          - {dataset: CERES-EBAF, project: OBS, type: sat, version: Ed4.2,
+             tier: 2, start_year: 2001, end_year: 2009, <<: *ref_tag}
+      ts:
+        variable: sst
+        <<: *var_rmse
+        <<: *era5_ref
+        preprocessor: rmse_ocean
+      ts_2:
+        variable: sst
+        short_name: ts
+        <<: *var_rmse
+        split: Ref2
+        preprocessor: rmse_ocean
+        additional_datasets:
+          - {dataset: HadISST, project: OBS, type: reanaly, version: 1, tier: 2, <<: *ref_tag}
+    scripts:
+      portrait:
+        script: portrait_plot.py
+        x_by: alias
+        y_by: variable  # use extra_facet since group and short_name don't work
+        group_by: project
+        normalize: "centered_median"
+        default_split: ERA5
+        nan_color: null
+        plot_kwargs:
+          vmin: -0.3
+          vmax: +0.3
+        cbar_kwargs:
+          label: Relative RMSE
+          extend: both
+
+  # perfmetrics_emd:  # seasonal normalized emd
```

Comment:
Please remove commented parts

Reply:
I removed the portrait plot recipe to avoid duplication with the recently merged recipe `recipe_portrait_CMIP.yml`.